### ISSUE TITLE
Does not work example code

### ISSUE THREAD

**Issue by hardikpatel043** [akitten/linkedin-public-profile-parser#38]
I have tried example code as below with express.

var express = require('express');
var app = express();
var LP = require('linkedin-public-profile-parser');
var url = 'https://www.linkedin.com/in/hardik-patel';

app.get('/', function (req, res) {

    LP(url, function (err, data) {
        res.send(JSON.stringify(data, null, 2));
    })
});

app.listen(3000);
console.log('api running');

Then when i hit url "http://localhost:3000" i get below result
{ "url": "https://www.linkedin.com/in/hardik-patel" }

Can anyone help me out?

**Comment by major13ua** [comment]
The same issue. It seems LinkedIn updated page structure again.

**Comment by iliasnaamane** [comment]
same issue here

**Comment by iteles** [comment]
Agreed @major13ua, that's probably the reason. If anyone on this thread has a little time to PR in the change, please do :blush:

**Comment by iliasnaamane** [comment]
@iteles @major13ua  the problem is not in the structure, the problem is that cheerio do not load the html variable in the profile.js line 12. You can test it when you remove the exception in line 7 lib/index.js an error cames up indicate that the html variable is undefined

**Comment by iliasnaamane** [comment]
I just saw that LinkedIn is blocking all requests form Amazon and other big providers to prevent web crawler getting their information, but it is not working in local too, please if anyone find out how to fix this issue, let us know, thanks

**Comment by vishwakarma** [comment]
I have raised a PR for same, 
the fix is mentioning accept-encoding header in fetcher.js for wreck
PR : https://github.com/akitten/linkedin-public-profile-parser/pull/39

**Comment by tim5go** [comment]
I created a modified version on https://github.com/tim5go/linkedin-crawler
I use `phantom` to add the page setting.
However, it seems that Linkedin will redirect you to a login page if you crawl it too frequently
